utilities/constants.py

```diff
@@ -88,8 +88,15 @@ class Protocols:
     HTTP: str = "http"
     HTTPS: str = "https"
     GRPC: str = "grpc"
-    TCP_PROTOCOLS: set[str] = {HTTP, HTTPS}
-    ALL_SUPPORTED_PROTOCOLS: set[str] = TCP_PROTOCOLS.union({GRPC})
+    REST: str = "rest"
+    TCP_PROTOCOLS: set[str] = {"HTTP", "HTTPS"}
+    ALL_SUPPORTED_PROTOCOLS: set[str] = TCP_PROTOCOLS.union({"GRPC"})
+
+
+class HTTPRequest:
+    # Use string formatting to set the token value when using this constant
+    AUTH_HEADER: str = "-H 'Authorization: Bearer {token}'"
```

Comment:
please also use in https://github.com/opendatahub-io/opendatahub-tests/blob/main/utilities/inference_utils.py#L197